### PR TITLE
Add country plan tab and country plan view

### PR DESCRIPTION
**Pull request by barshathakuri**
Addresses: https://github.com/IFRCGo/go-frontend/issues/2435

Depends on: https://github.com/IFRCGo/go-api/pull/1588

### CODE REVIEW

File: src/root/components/TextOutput/index.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ interface BaseProps {
   labelContainerClassName?: string;
   description?: React.ReactNode;
   descriptionContainerClassName?: string;
-  valueContainerClassName?: string;
+  valueContainerClassName?:  React.ReactNode;
```
Comment (frozenhelium) [inline src/root/components/TextOutput/index.tsx:17]: This is probably a mistake

File: src/root/views/region.js [hunk 2]
```diff
@@ -77,25 +77,26 @@ class AdminArea extends SFPComponent {
...
 
-  getTabDetails () {
+  getTabDetails() {
     const { strings } = this.context;
     return [
       { title: strings.regionOperationsTab, hash: '#operations' },
       { title: strings.region3WTab, hash: '#3w' },
       { title: strings.regionRiskTab, hash: '#risk-watch' },
       { title: strings.regionProfileTab, hash: '#regional-profile' },
+      { title: strings.regionCountryPlansTab, hash: '#country-plans' },
```
Comment (frozenhelium) [inline src/root/views/region.js:87]: We should remove this as well. Also, this file only contains the white-space changes (probably auto-formatted during the save), please be careful of such changes.

**Review by AdityaKhatri — APPROVED** [review]

**Comment by frozenhelium** [comment]
Let's wait for the back-end to be ready!

**Review by frozenhelium — APPROVED** [review]

**Comment by frozenhelium** [comment]
@szabozoltan69 please also merge this after https://github.com/IFRCGo/go-api/pull/1588 and deploy to the staging. Thanks! :)

**Comment by szabozoltan69** [comment]
@frozenhelium Deployed to staging (after go-api was deployed also). :-)

**Pull request merged**